en/core-libraries/behaviors/tree.rst

```diff
@@ -453,6 +453,24 @@ are a few more tree-orientated permutations at your disposal.
           [5] =>  "_Extreme fishing"
       )
 
+    .. php:method:: formatTreeList($results, $options=array())
+
+    .. versionadded:: 2.7
+
+    :param $results: Results of a find('all') call.
+    :param $options: Options to pass into.
+
+    This method will return data similar to
+    :ref:`model-find-list` but with a nested prefix that is specified
+    in the ``spacer`` option to show the structure of your data.
+    See generateTreeList() for an example output.
+
+    Supported options are:
+
+    * ``keyPath``: A string path to the key, i.e. "{n}.Post.id".
+    * ``valuePath``: A string path to the value, i.e. "{n}.Post.title".
+    * ``spacer``: The character or characters which will be repeated.
+
```

Comment:
I think it is good to show how to use it in combination with results from another find